app/requests/remote/user.ts

```diff
@@ -378,3 +339,39 @@ export const getSessions = async (currentUserId: string) => {
         await forceLogoutIfNecessary(e);
     }
 };
+
+type LoadedUser = {
+    currentUser?: RawUser,
+    error?: ErrorApi
+}
+
+export const ssoLogin = async (serverUrl: string) => {
+    let deviceToken;
+
+    const {error, defaultDatabase} = await getDefaultDatabase();
```

Comment:
I believe at this point the default database should already be there so no async/await should be needed. With that in mind, why would the default database not be present at this point ?